yb-voyager/src/tgtdb/target_db_interface.go

```diff
@@ -43,6 +43,10 @@ type TargetDB interface {
 	RestoreSequences(sequencesLastValue map[string]int64) error
 	GetImportedEventsStatsForTable(tableName string, migrationUuid uuid.UUID) (*EventCounter, error)
 	GetImportedSnapshotRowCountForTable(tableName string) (int64, error)
+	GetGeneratedAlwaysIdentityColumnNamesForTable(table string) ([]string, error)
+	GetGeneratedAlwaysAsIdentityColumnsForTables(tables []string) (map[string][]string, error)
```

Comment:
Nit: Following parts of the name should be present in both the names: `As` and `Names`